recaptcha_enterprise/demosite/app/templates/home.html

```diff
@@ -15,66 +15,96 @@
+  <!-- See: https://cloud.google.com/recaptcha-enterprise/docs/create-assessment -->
+  <script type="text/javascript">
+    async function viewHomepage({ token }) {
+      // Include the token for server-side assessment.
+      const body = {
+        token,
+      };
+      // Code for fetching the assessment from server-side goes here.
+      // Refer to demo app backend code for more information.
+      // If you already use a library or framework for event handlers, check if this
+      // call can be accommodated the usual way.
+      const score = await fetchServerResponse({
+        body,
+        url: "on_homepage_load",
+      });
+      // In this demo, the assessment score is displayed in the client.
+      // But generally, you should avoid using the assessment response in the
+      // client and handle it on the server-side.
+      useAssessmentInClient(score);
+    }
+  </script>
+  <!-- ATTENTION: reCAPTCHA Example (Client Part 1 of 2) Ends -->
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <meta
+      name="description"
+      content="Explore the different ways of using reCAPTCHA Enterprise to protect web sites and applications."
+  />
+  <link
       href="https://fonts.googleapis.com/css?family=Press+Start+2P"
       rel="stylesheet"
-    />
-    <link
+  />
+  <link
       href="https://fonts.googleapis.com/css?family=Material+Icons&display=block"
       rel="stylesheet"
-    />
-    <link href="../static/styles/global.css" rel="stylesheet" />
-    <script type="module" src="../static/scripts/component.js"></script>
-    <script type="text/javascript" src="../static/scripts/global.js"></script>
-    <!-- ATTENTION: reCAPTCHA Example (Client Part 1/1) Starts -->
-    <!-- See: https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#page-load -->
-    <script src="https://www.google.com/recaptcha/enterprise.js?render={{site_key}}"></script>
-    <script type="text/javascript">
-      grecaptcha.enterprise.ready(async () => {
-        const token = await grecaptcha.enterprise.execute(
-            "{{site_key}}",
-            {
-              action: "home",
-            }
-        );
-        await homepage({token});
-      });
-    </script>
-    <!-- ATTENTION: reCAPTCHA Example (Client Part 1/1) Ends -->
-  </head>
-  <body>
-    <recaptcha-demo step="home"></recaptcha-demo>
-  </body>
+  />
+  <link
+      rel="icon"
+      type="image/x-icon"
+      href="../static/favicon-e9e3fe3130a875eb.ico"
+  />
+  <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="../static/favicon-16x16-e9e3fe3130a875eb.png"
+  />
+  <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="../static/favicon-32x32-e9e3fe3130a875eb.png"
+  />
+  <link
+      rel="icon"
+      type="image/png"
+      sizes="48x48"
+      href="../static/favicon-48x48-e9e3fe3130a875eb.png"
+  />
+  <link href="../static/global-15fca5ccf020c02b.css" rel="stylesheet" />
```

Comment:
Are these filenames WAI?

Reply:
They're generated, presumably for version control? @morewry Any thoughts here?

Reply:
@morewry PTAL

Reply:
Yep. It's a content hash that changes if the contents of the file do; it doesn't serve all the usual purposes in this context, but it adds some version information useful for debugging. The important part is that the browser can load the files and they result in the right UI.